src/domains/citizen-auth-common/03_storage.tf

```diff
@@ -243,3 +243,9 @@ resource "azurerm_storage_table" "profile_emails" {
   name                 = "profileEmails"
   storage_account_name = module.io_citizen_auth_storage.name
 }
+
+resource "azurerm_storage_queue" "profiles_to_sanitize" {
+  depends_on           = [module.io_citizen_auth_storage, azurerm_private_endpoint.table]
+  name                 = "profiles-to-sanitize"
+  storage_account_name = module.io_citizen_auth_storage.name
+}
```

Comment:
We need to add the private endpoint for the storage queue (see [here](https://github.com/pagopa/io-infra/blob/ee7f762e96a0f9a05825a4da3cb9088bbfcac663/src/domains/citizen-auth-common/03_storage.tf#L58-L59))